activities.json

```diff
@@ -696,16 +696,16 @@
     "url": "https://w3c.github.io/mst-content-hint/"
   },
   {
-    "ciuName": "native-filesystem-api",
-    "description": "This document defines a web platform API that lets websites gain write access to the native file system. It builds on [FILE-API], but adds lots of new functionality on top.",
-    "id": "native-file-system",
+    "ciuName": "file-system-access-api",
```

Comment:
I don't see this on caniuse.

Reply:
Pending PR: https://github.com/Fyrd/caniuse/pull/5622